client/src/components/pages/application/Profile/index.js

```diff
@@ -1,5 +1,329 @@
+                      {data.email}
+                    </Typography>
+                  </td>
+                </tr>
+                <tr>
+                  <td className="item">
+                    <Typography variant="body2" align="left">
+                      Gender
+                    </Typography>
+                  </td>
+                  <td>
+                    <Typography variant="body2" align="left">
+                      {data.gender}
+                    </Typography>
+                  </td>
+                </tr>
+                <tr className="doted">
+                  <td className="item">
+                    <Typography variant="body2" align="left">
+                      Age
+                    </Typography>
+                  </td>
+                  <td>
+                    <Typography variant="body2" align="left">
+                      {data.age}
+                    </Typography>
+                  </td>
+                </tr>
+                <tr className="doted">
+                  <td>
+                    <Typography variant="body2" color="primary" align="left">
+                      Accounts
+                    </Typography>
+                  </td>
+                  <td />
+                </tr>
+                <tr>
+                  <td className="item">
+                    <Typography variant="body2" align="left">
+                      Github
+                    </Typography>
+                  </td>
+                  <td>
+                    <Typography variant="body2" align="left">
+                      {data.githubLink}
+                    </Typography>
+                  </td>
+                </tr>
+                <tr>
+                  <td className="item">
+                    <Typography variant="body2" align="left">
+                      FreeCode Camp
+                    </Typography>
+                  </td>
+                  <td>
+                    <Typography variant="body2" align="left">
+                      {data.freeCodeCampLink}
+                    </Typography>
+                  </td>
+                </tr>
+                <tr className="doted">
+                  <td className="item">
+                    <Typography variant="body2" align="left">
+                      Code Wares
+                    </Typography>
+                  </td>
+                  <td>
+                    <Typography variant="body2" align="left">
+                      {data.codeWarsLink}
+                    </Typography>
+                  </td>
+                </tr>
+                <tr className="doted">
+                  <td>
+                    <Typography variant="body2" color="primary" align="left">
+                      Technical Task
+                    </Typography>
+                  </td>
+                  <td />
+                </tr>
+                {Technical &&
+                  Technical.map(({ taskName }) => (
```

Comment:
```suggestion
                  Technical.map(({ taskName, _id }) => (
                      <tr key={_id}>
```
fix this line
I know you, It's a simple issue and you probably knew about this.